pkg/operator/bootstrap.go

```diff
@@ -29,6 +30,7 @@ func RenderBootstrap(
 ) error {
 	filesData := map[string][]byte{}
 	files := []string{
+		additionalTrustBundleFile,
```

Comment:
the trust bundle file will only be present if it was required by the user. this is optional..

Reply:
ack makes sense

Reply:
fixed